source/tutorials/thermal-daemon/thermal-daemon.rst

```diff
@@ -51,6 +64,15 @@ KabyLake NUC
 AppoloLake NUC
 ~~~~~~~~~~~~~~
 
+- Platform Thermal Parameters
+
+================ =========================
+Tjmax 	         105 (degree celsisus)
+Thermal Zone     x86_pkg_temp
+Cooling device   soc power limit 1
+================ =========================
+
+- Thermal Management Policy
```

Comment:
The bullet list must be ended with an empty line. You have to add a blank line below.